### DISCUSSION TITLE
UX Designer

### DISCUSSION THREAD

**Discussion by DenizBagheri** [WeAreMahsaAmini/Core#208] [Introductions]
Hello,

I'm a UX designer with focus on Service Design. Would love to help out. Funny coincidence, we were discussing with some friends about a very similar idea. So happy to see some people has made is happen. I'm living in Munich, Germany.

**Comment by ehsaaanes** [comment]
Hi
I'm looking for people who can help creating an app for donation. 
I’d appreciate it if you check it here #163. 
please write a comment or let me know what you think about it. 
So much thanks*
